kitsune/users/models.py

```diff
@@ -256,6 +256,31 @@ def last_contribution_date(self):
     def settings(self):
         return self.user.settings
 
+    @property
```

Comment:
Also, since this method will be making a DB query, I would use `functools.cached_property` instead.

Reply:
I couldn't find a good way to make this change to cached_property exist with tests, which made me think maybe we shouldn't use cached_property here - user content seems like something that is apt to change regularly, in real time. I think the caching is only for the lifetime of the object, so maybe wouldn't be a problem outside of tests, but maybe it would? I know I could always just handle the cache in the tests (so, not building a solution to meet tests - just inspired to ask the question once I saw that issue during testing). Thoughts?